ha/const.py

```diff
@@ -74,6 +74,8 @@
 PCS_FAILCOUNT_STATUS="pcs resource failcount show"
 PCS_STATUS = "pcs status"
 PCS_CLUSTER_DESTROY="pcs cluster destroy"
+CIB_FILE="/var/log/seagate/cortx/ha/cortx-lr2-cib.xml"
```

Comment:
cortx-lr2-cib.xml -> cortx-r2-cib.xml.
This is not lyve rack.

Reply:
Done with change will push in next patch

Reply:
Done